float64[3] p
float64[3] p_d
float64[3] err_p
float64[3] v
float64[3] v_d
float64[3] err_v
float64[3] a
float64[3] a_d
float64 sumfootforce 
float64 d_mp
float64[3] error_1
float64[3] error_2
float64[3] integral
float64[3] tau_t
float64[3] template_sgn
float64[3] c_error_1
float64[3] c_error_2
float64[3] c_integral
float64[3] c_tau_t
float64[3] c_template_sgn